### ISSUE TITLE
Navbar spacing

### ISSUE THREAD

**Issue by GatesDupont** [chrisyeh96/chrisyeh96.github.io#6]
First of all, this is a great take on the Cayman theme, thanks so much! I was wondering about spacing in the navbar -- I checked developer tools and there seems to be an empty element propagated between About and Projects. Alone, this is fine, but I'd like to add more sections (a coursework tab, etc.), but this puts emphasis on the empty element causing additional spacing. I think this stems from _includes/navbar.html. However, I've read through it a few times and can't figure out where its coming from. Any suggestions on removing this? Thanks!

**Comment by chrisyeh96** [comment]
Good catch! The extra spacing was due to an invisible link back to the home page. In Jekyll, `site.pages` includes the main `index.md` page, which I forgot to filter out. My `index.md` page doesn't have a `page.title` attribute, so it never displayed a visible link and therefore I didn't notice there was an extra space until you pointed it out. In commit 99ab6a1b, I fixed this by adding a filter to only include pages with a non-nil `page.title` attribute.
